airflow/www/static/js/dag/details/dagRun/ClearRun.tsx

```diff
@@ -59,31 +60,67 @@ const ClearRun = ({ runId, ...otherProps }: Props) => {
     onQueue({ confirmed: true });
   };
 
-  useKeysPress(keyboardShortcutIdentifier.dagRunClear, clearExistingTasks);
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [doNotShowAgain, setDoNotShowAgain] = useState(false);
+
+  useEffect(() => {
+    const storedValue = localStorage.getItem("doNotShowAgain");
+    if (storedValue) {
+      setDoNotShowAgain(JSON.parse(storedValue));
+    }
+  }, []);
```

Comment:
This doesn't need to be in `useEffect` we can just check this in the line before our useState and then set it as the default.